rfcs/convergence/first-rule-of-aria.md

```diff
@@ -0,0 +1,164 @@
+- Sometimes live regions are used to make up for differences between native behavior and custom ARIA widgets. This technique usually falls spectacularly flat on braille displays, where live regions have notably flaky support, particularly when they conflict with interaction.
+
+The difficulty replicating the robust nature of HTML with custom scripting and ARIA is why the [First Rule of ARIA](https://w3c.github.io/using-aria/#rule1) is "don't use ARIA".
+
+## Detailed Proposal
+
+The proposal is a pattern in favor of using semantic elements as a default (with opt-out mechanisms) for convergence components. This is particularly important for interactive elements like buttons, links, inputs, radios, etc.
+
+This proposal does not introduce any API, is just a pattern to be followed when creating converged components, and valid to add that in extreme cases, like table, that semantic first approach is simply not viable, than this should be ignored.
+
+Converged components should follow these steps:
+
+---
+
+If is equivalent to native element - `Button`, `Link`, `Select`
+
+- Should use native element by default
+- Should implement [as-props](./as-props.md) to opt-out native element
+- Should limit possible elements to opt-out (e.g: `Button` should opt-out from `button` for `div` and `span`)
+- In case of opting-out element should implement required ARIA event listeners and attributes, such as `role`
+- Provide authors an option to use a semantic HTML version (e.g. Dropdown should have an author option to use a `<select>` element, even if we also allow a custom version with a stylable extendable options list).
+
+##### Button Example
+
+```tsx
+<Button>This is a Simple Button</Button>
+<Button as="div">This is a div that looks and behaves as a Button</Button>
+<Button as="div" role={undefined} tabIndex={undefined}>
+  This is a div that looks like Button but doesn't behave as such
+  therefore, I can add a <Link to="/somewhere">Link inside of it</Link>
+</Button>
+```
+
+```html
+<!-- Semantic version -->
+<button class="button-class">This is a Simple Button</button>
+<!-- Aria version -->
+<div class="button-class" tabindex="0" role="button">This is a div that looks and behaves as a Button</div>
+<!-- Style only -->
+<div class="button-class">
+  This is a div that looks like Button but doesn't behave as such therefore, I can add a
+  <a href="/somewhere">Link inside of it</a>
+</div>
+```
+
+---
+
+If isn't equivalent to native elements but can benefit from using native elements in it's implementation - `Accordion`, `Carousel`, `Disclosure`, `MenuButton`, `Menu`, `MenuBar`, `Tabs`
+
+- Should treat internal elements that could follow previous category as such (e.g: `Accordion` has an internal element with role `button`, this element should previous category)
```

Comment:
I would say that you should just move accordion to the previous category, in this case there **is a native** element for that role

Reply:
do you mean details/summary?

Reply:
Ah sorry wasn't clear enough, my point was that `Accordion` design pattern actually does explicitly call for the use of a `button` element, so it doesn't exactly fit the bill of 'can use native element' rather 'should use native element'